package.json

```diff
@@ -19,6 +19,8 @@
     "prebuild": "cp webpack.config.prod.js node_modules/react-scripts/config",
     "postbuild": "npx uglify-es build/static/js/main.*.js -o build/static/js/$(ls build/static/js | head -n 1)",
     "test": "react-scripts test --env=jsdom",
+    "deploy-s3": "npm run delete-source-maps && aws s3 sync build/ s3://explorer.chlu.io --delete --exclude index.html --acl public-read && aws s3 sync build/ s3://explorer.chlu.io --exclude '*' --include index.html --acl public-read --cache-control 'max-age=0'",
+    "delete-source-maps": "find ./build -name *.map -type f -delete",
```

Comment:
Instead of deleting source maps before syncing this to S3, should we consider not generating source maps for production builds? Or is this a "safety valve"?

Reply:
The build process is handled by react-scripts so not sure if we can avoid generating the source maps

Reply:
Yeah, interesting approach they took https://github.com/facebook/create-react-app/pull/2742#issuecomment-313658034